File: v2/java/src/main/java/vestorly/client/model/NewsletterSettingsInput.java
package vestorly.client.model;

import vestorly.client.model.NewsletterSetting;

import com.wordnik.swagger.annotations.*;
import com.fasterxml.jackson.annotation.JsonProperty;


@ApiModel(description = "")
public class NewsletterSettingsInput  {
  
  private NewsletterSetting newsletterSetting = null;

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("newsletter_setting")
  public NewsletterSetting getNewsletterSetting() {
    return newsletterSetting;
  }
  public void setNewsletterSetting(NewsletterSetting newsletterSetting) {
    this.newsletterSetting = newsletterSetting;
  }

  

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class NewsletterSettingsInput {\n");
    
    sb.append("  newsletterSetting: ").append(newsletterSetting).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}

File: v2/csharp/src/main/csharp/vestorly/Model/NewsletterSettingsInput.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;
using System.Runtime.Serialization;

namespace vestorly.model {
  [DataContract]
  public class NewsletterSettingsInput {
    
    
    [DataMember(Name="newsletter_setting", EmitDefaultValue=false)]
    public NewsletterSetting newsletterSetting { get; set; }

    
    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class NewsletterSettingsInput {\n");
      
      sb.Append("  newsletterSetting: ").Append(newsletterSetting).Append("\n");
      
      sb.Append("}\n");
      return sb.ToString();
    }
  }
  
  
}
